package.json

```diff
@@ -42,7 +42,8 @@
     "pug-lint": "pug-lint lib/plugins/html/templates",
     "test": "mocha",
     "check-licenses": "tools/check-licenses.js",
-    "travis": "npm run lint && npm run test"
+    "travis": "npm run lint && npm run test",
+    "build:css": "node-sass lib/plugins/html/src/sass/main.scss > lib/plugins/html/assets/css/index.css && cleancss -o lib/plugins/html/assets/css/index.min.css lib/plugins/html/assets/css/index.css"
```

Comment:
No worries. You could add " && npm run build:css" to the travis script. It's always helpful to have Travis execute stuff to catch missing dependencies.